### PR TITLE
Another fix to example.

### PR DESCRIPTION
**Pull request by jaeilepp**

### CODE REVIEW

**Comment by Eric89GXL** [comment]
Thanks, this was my bad. Feel free to push with `[circle full]` in the
commit message to see if there are subsequent failures

**Comment by agramfort** [comment]
LGTM

**Comment by jaeilepp** [comment]
Ready for review.

**Comment by agramfort** [comment]
LGTM 

Please merge and backport 

Thx

File: examples/datasets/plot_spm_faces_dataset.py [hunk 1]
```diff
@@ -74,7 +74,7 @@
 
 evoked = [epochs[k].average() for k in event_ids]
 
-contrast = evoked[1] - evoked[0]
+contrast = combine_evoked(evoked, weights=[-1, 1])  # Faces - scrambled
```
Comment (wmvanvliet) [inline examples/datasets/plot_spm_faces_dataset.py]: actually, I find the original code much clearer in this instance.

Comment (jaeilepp) [inline examples/datasets/plot_spm_faces_dataset.py]: But it won't work. Subtraction is deprecated.

Comment (wmvanvliet) [inline examples/datasets/plot_spm_faces_dataset.py]: ah yes.

**Pull request merged**